socorro/external/postgresql/adi.py

```diff
@@ -51,19 +51,40 @@ def get(self, **kwargs):
                 product_adu
             LEFT OUTER JOIN product_versions pv USING (product_version_id)
             WHERE
-                pv.product_name = %(product)s
-                AND pv.version_string IN %(versions)s
-                AND os_name IN %(platforms)s
-                AND adu_date BETWEEN %(start_date)s AND %(end_date)s
+                pv.product_name = %%s
+                AND (%s)
+                AND os_name IN %%s
+                AND adu_date BETWEEN %%s AND %%s
```

Comment:
I did that in the old advanced search back-end code, and it was quite ugly. But there were a lot of them scattered in different places, so maybe for something small like here it could be OK. Anyway, I agree that dict > tuple.